plugins/hubspot/src/blog.ts

```diff
@@ -46,10 +72,15 @@ export async function getBlogPluginContext(): Promise<BlogPluginContext> {
     const collection = await framer.getManagedCollection()
     const collectionFields = await collection.getFields()
 
-    const rawExcludedFieldIds = await collection.getPluginData(PLUGIN_EXCLUDED_FIELD_IDS)
-    const excludedFieldIds = parseJsonToArray<string>(rawExcludedFieldIds)
+    const allPossibleFieldIds = HUBSPOT_BLOG_FIELDS.map(field => field.id)
+
+    const rawExcludedFieldHash = await collection.getPluginData(PLUGIN_EXCLUDED_FIELD_HASH)
 
-    if (!excludedFieldIds) {
+    const includedFieldIds = collectionFields.map(field => field.id)
+    const excludedFieldIds = allPossibleFieldIds.filter(id => !includedFieldIds.includes(id))
+    const excludedFieldHash = simpleHash(excludedFieldIds.join(","))
```

Comment:
Please, join like in Google Sheets, with something reasonably unique. It's unlikely to cause problems in blogs, but it's better to do it the same way everywhere. Should probably be a tiny utility function, used by both blogs and HubDB.

Probably also best to sort the ids, as to not rely on HubSpot's ordering.